package message

import "fmt"

var ErrMaxAttemptsExceeded = fmt.Errorf("max attempts exceeded")
